atlp-rwanda/e-comm-team-techsmith-fn
# #185505204 Bug fixes for Pre-demo 29/06/2023

## What does this PR do?

This PR fixes bugs adhering to feedbacks ahead of our final pre-demo of 29th June 2023.

## Description of Task to be completed?

1. Refine route outlets
2. Fix buyer checkout workflow
3. Fix categories page
4. Refine user authentication experience
5. Fix admin dashboard
6. Update product search page
7. Ensure font-sizes consistency
8. Fix seller dashboard workflows

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bg-auth-purchases-185505204`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## Any background context you want to provide?

This PR will feature all bug fixes required ahead of the demo. Other discoveries will be addressed using this same PR.

## What are the relevant pivotal tracker/Trello stories?

+ [#185505204](https://www.pivotaltracker.com/story/show/185505204)
